graphql-server/go-server/graph/model/models_gen.go

```diff
@@ -6,6 +6,10 @@ import (
 	"time"
 )
 
+type PageNode interface {
+	IsPageNode()
+}
```

Comment:
How about we place `models_gen` inside `generated` since this is auto-generated by gqlgen too.